testing/py-google-apputils/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-google-apputils
+_pkgname=google-apputils
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Google Application Utilities for Python"
+url="https://github.com/google/google-apputils"
+arch="noarch"
+license="Copyright 2014, Google Inc. All rights reserved."
+depends="python"
```

Comment:
google-apputils [has some Python dependencies](https://github.com/google/google-apputils/blob/master/setup.py#L28-L32) that must be also specified in `depends`.